cmd/root.go

```diff
@@ -109,6 +110,57 @@ application configuration to remote clusters.
 
 		return nil
 	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cobra.NoArgs(cmd, args)
+		}
+
+		pluginName := args[0]
+		_, err := plugin.Find(appFs, pluginName)
+		if err != nil {
+			return cobra.NoArgs(cmd, args)
+		}
+
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Help()
+		}
+		pluginName, args := args[0], args[1:]
+		p, err := plugin.Find(appFs, pluginName)
+		if err != nil {
+			return err
+		}
+
+		return runPlugin(p, args)
+	},
+}
+
+func runPlugin(p plugin.Plugin, args []string) error {
```

Comment:
What's your opinion on putting the this plugin function to the new `plugin` package?

Reply:
It does belong there, but the information I wanted to share isn't easily accessible from there. Need to go back and make that easier to retrieve. Then it can move.